ios/Classes/NetIamyellowTiwsWSProxy.m

```diff
@@ -62,6 +62,9 @@ -(void)webSocket:(SRWebSocket*)webSocket didCloseWithCode:(NSInteger)code reason
 
 -(void)webSocket:(SRWebSocket*)webSocket didReceiveMessage:(id)data
 {
+    if ([data isKindOfClass:[NSData class]])
```

Comment:
Thank you for the PR.
Would you mind enclosing all if and else blocks in opening/closing braces for consistency? (see also lines 128-131 of NetIamyellowTiwsWSProxy.m)